kythe/extractors/bazel/extract.sh

```diff
@@ -66,6 +66,13 @@ fi
 # care if some targets fail, but bazel will return a failure code if any
 # targets fail.
 /kythe/bazelisk --bazelrc=/kythe/bazelrc "$@" || true
+retval=$?
+if [ $retval -eq 1 ]; then
+    echo "Not all bazel targets built successfully, but continuing anyways."
+else if [ $retval -ne 0 ];
```

Comment:
You likely want to use the builtin `[[ ... ]]` or `(( ... ))` for these

Reply:
:+1: